docs/skopeo.1.md

```diff
@@ -136,9 +135,9 @@ Verify a signature using local files, digest will be printed on success.
 show help for `skopeo`
 
 # FILES
-  **/etc/containers/policy.json**
-  Default signature verification policy file, if **--policy** is not specified.
-  The policy format is documented in https://github.com/containers/image/blob/master/docs/policy.json.md .
+  **/etc/containers/trust.json**
+  Default signature verification trust file, if **--trust** is not specified.
+  The trust format is documented in https://github.com/containers/image/blob/master/docs/trust.json.md .
```

Comment:
Another external reference, https://github.com/containers/image/pull/50 needs to be updated.